### DISCUSSION TITLE
Will there be a new version of syspass that works with PHP 8.1?

### DISCUSSION THREAD

**Discussion by eager-2-learn** [nuxsmin/sysPass#1929] [General]
The current version of syspass works with PHP 7.3 and 7.4
But Ubuntu 20 works with PHP 8.1
Will there be a new version of syspass that works with PHP 8.1 en cab be used on Ubuntu 20?

**Comment by jonastandinco** [comment]
I am also trying to get this to work on PHP 8. PHP 7 is end of life already. Considering the sensitivity of this system, I think it should now support PHP 8 version and onwards. Is there any news if this is in the works?

**Comment by patrice-go-dsi** [comment]
I am using sysPass with php 8.1 and it works.

i had to hack the composer.json file with :

`
>     "phpunit/phpunit": "^9.3",
>     "misantron/dbunit": "^5",
`

And also the file lib/SP/Util/checks.php : 
`
public static function checkPhpVersion()
{
    return version_compare(PHP_VERSION, '7.3', '>=')
        && version_compare(PHP_VERSION, '9.0', '<');
}
`
